### PR TITLE
Update pub/sub subscriptions

### LINKED ISSUE #782: Update docs for declarative pub/sub

### PR DESCRIPTION
**Pull request by yaron2**
Closes #782.

### CODE REVIEW

File: howto/consume-topic/README.md [hunk 1]
```diff
@@ -34,6 +34,50 @@ To deploy this into a Kubernetes cluster, fill in the `metadata` connection deta
 
 ## Subscribe to topics
 
+Dapr allows two methods by which you can subscribe to topics: programatically and declaratively.
```
Comment (mchmarny) [inline howto/consume-topic/README.md:37]: Consider sharper contrast and active voice. For example: 

Dapr supports two methods for creating topic subscription:

* programmatic - where the intend to subscribe to specific topic is defined in user code
* declarative - where the subscription is defined through external configuration

Comment (mchmarny) [reply howto/consume-topic/README.md:37]: And link these to the corresponding sections

Comment (yaron2) [reply howto/consume-topic/README.md:37]: The sections are right under that sentence..

**Review by mchmarny — COMMENTED** [review]
nit on intro, otherwise lgtm

File: howto/consume-topic/README.md (hunk 1 above)
Comment (yaron2) [reply howto/consume-topic/README.md]: > Consider sharper contrast and active voice. For example:
> 
> Dapr supports two methods for creating topic subscription:
> 
> * programmatic - where the intend to subscribe to specific topic is defined in user code
> * declarative - where the subscription is defined through external configuration

Changed.

**Review by mchmarny — APPROVED** [review]
/lgtm

**Pull request merged**